public/app/features/admin/UserAdminPage.tsx

```diff
@@ -0,0 +1,163 @@
+import { UserPermissions } from './UserPermissions';
+import { UserSessions } from './UserSessions';
+import { UserLdapSyncInfo } from './UserLdapSyncInfo';
+import { StoreState, UserDTO, UserOrg, UserSession, SyncInfo } from 'app/types';
+import {
+  loadUserProfile,
+  loadUserOrgs,
+  loadUserSessions,
+  revokeSession,
+  revokeAllSessions,
+  loadLdapSyncStatus,
+} from './state/actions';
+import { UserOrgs } from './UserOrgs';
+
+interface Props {
+  navModel: NavModel;
+  userId: number;
+  user: UserDTO;
+  orgs: UserOrg[];
+  sessions: UserSession[];
+  ldapSyncInfo: SyncInfo;
+
+  loadUserProfile: typeof loadUserProfile;
+  loadUserOrgs: typeof loadUserOrgs;
+  loadUserSessions: typeof loadUserSessions;
+  loadLdapSyncStatus: typeof loadLdapSyncStatus;
+  revokeSession: typeof revokeSession;
+  revokeAllSessions: typeof revokeAllSessions;
+}
+
+interface State {
+  isLoading: boolean;
+}
+
+export class UserAdminPage extends PureComponent<Props, State> {
+  state = {
+    isLoading: true,
+  };
+
+  async componentDidMount() {
+    const { userId, loadUserProfile, loadUserOrgs, loadUserSessions, loadLdapSyncStatus } = this.props;
+    try {
+      await loadUserProfile(userId);
+      await loadUserOrgs(userId);
+      await loadUserSessions(userId);
+      await loadLdapSyncStatus();
+    } finally {
+      this.setState({ isLoading: false });
+    }
+  }
+
+  handleUserUpdate = (user: UserDTO) => {
+    console.log('update user', user);
+  };
+
+  handleUserDelete = (userId: number) => {
+    console.log('delete user', userId);
+  };
+
+  handleUserDisable = (userId: number) => {
+    console.log('disable user', userId);
+  };
+
+  handleGrafanaAdminChange = (isGrafanaAdmin: boolean) => {
+    console.log('handleGrafanaAdminChange', isGrafanaAdmin);
+  };
+
+  handleOrgRemove = (orgId: number) => {
+    console.log('handleOrgRemove', orgId);
+  };
+
+  handleOrgRoleChange = (orgId: number, newRole: string) => {
+    console.log('handleOrgRoleChange', orgId, newRole);
+  };
+
+  handleSessionRevoke = (tokenId: number) => {
+    const { userId, revokeSession } = this.props;
+    revokeSession(tokenId, userId);
+  };
+
+  handleAllSessionsRevoke = () => {
+    const { userId, revokeAllSessions } = this.props;
+    revokeAllSessions(userId);
+  };
+
+  handleUserSync = () => {
+    console.log('sync user', this.props.user.login);
+  };
+
+  render() {
+    const { navModel, user, orgs, sessions, ldapSyncInfo } = this.props;
+    const { isLoading } = this.state;
+    const isLDAPUser = user && user.isExternal && user.authLabels && user.authLabels.includes('LDAP');
+
+    return (
+      <Page navModel={navModel}>
+        <Page.Contents isLoading={isLoading}>
+          {user && (
+            <>
+              <UserProfile
+                user={user}
+                onUserUpdate={this.handleUserUpdate}
+                onUserDelete={this.handleUserDelete}
+                onUserDisable={this.handleUserDisable}
```

Comment:
The react style guide we follow name handlers the same as the callback props (so handleUserUpdate should be named onUserUpdate)

Reply:
Ok, I just used official React style guide where handlers should use different prefix (`handle` instead of `on`).